docs/testbenches/project_based/template/index.rst

```diff
@@ -8,33 +8,44 @@ Project based test bench template
 Overview
 -------------------------------------------------------------------------------
 
+**\*This section must contain: the purpose of the testbench/ type of interface
+it's validating, the corresponding hdl project and the HDL Github documentation
+page.**\ \*
+
 Block design
 -------------------------------------------------------------------------------
 
+**\*Mention the HDL and SV components of the testbench .**\ \*
+
 Block diagram
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-..
-   MUST: Use SVG format for the diagram
+If the testbench has multiple ways of configuration, then make subsections to
+this section and show the default configuration and some other popular modes.
```

Comment:
If the testbench has multiple configurations that change the block design itself, then make subsections for each of these configuration-block design pairs. Ex: project is configured in serial mode or parallel mode. 
If there are many configuration options on how the block design looks, try to find ways to generalize them. Ex: multiple interfaces of the same type are enabled by a parameter.

Reply:
Modified the text according to the suggestion.